resolwe_bio/docker_images/latex/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM resolwe/base:ubuntu-16.04
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
```

Comment:
Any reason for running `upgrade` here? I think all upgrades should only be handled in base images.